autocertkit/network_tests.py

```diff
@@ -1109,3 +1109,195 @@ class GROOffloadBridgeTestClass(GROOffloadTestClass):
     GRO is on by default from XS 6.5 """
     network_backend = "bridge"
     order = 5
+
+
+class SRIOVTest(IperfTest):
```

Comment:
Its not clear to me why a different implementation is required for starting Iperf in a VM which uses SR-IOV. Shouldn't the tests be agnostic? We are still starting an iperf server on a given interface, and running an iperf command from a given client.

Reply:
I have same idea to you before.

You can see MulticastTest derived from IperfTest as well.
```
class MulticastTest(IperfTest):
```
When I saw MulticastTest I felt it's short to read/understand indeed.

The reason I add SRIOVTest class are 
- I have different iperf client option (with -B) against default of IperfTest
```
cmd_str = 'iperf -c %s -u -B %s' % (self.server_ip, self.client_ip)
```
- I prefer to let iperf test log.debug() be with specific 'SR-IOV' labels, e.g.
```
log.debug('Starting SR-IOV IPerf server')
log.debug('IPerf log on SR-IOV IPerf server: %s' % cmd_result)
'Error: SR-IOV IPerf test failed. IPerf data: %s' % cmd_result)
```

Reply:
Understood - but in which case I'd presume we'd be better off modifying the base class to have several class variables rather than having to reimplement a bunch of functions which are mostly doing similar things.

Reply:
Agree with you. Some to network config file review, I think I am able to uniform them later.

Reply:
It's fine not to fix the multicast case - focus on making the changes for the SR-IOV test case first.

Reply:
I do not mean that I won't refactor Multicast test. I mean I will make Both SR-IOV and Multicast tests  to use IperfTest class directly, and currently the only concern is that I have no enough time to do this as the deadline is coming, you know this point is not the most important factor for SR-IOV feature. I hope the code is able to be merged ASAP, so that QA can make test early.